.circleci/config.yml

```diff
@@ -622,6 +622,17 @@ jobs:
           name: "Build and test"
           command: build sequencer-client
 
+  canary:
+    docker:
+      - image: aztecprotocol/alpine-build-image
+    resource_class: small
+    steps:
+      - *checkout
+      - *setup_env
+      - run:
+          name: "Build"
+          command: cond_spot_run_build canary-build 32
```

Comment:
Do we need a spot machine to build the canary, or would a regular ubuntu large do it?

Reply:
Turns out we don't.